### PR TITLE
Beep Boop

### PR DESCRIPTION
**Pull request by Fortifier42**

### CODE REVIEW

File: src/main/java/net/aufdemrand/denizen/tags/core/ServerTags.java [hunk 1]
```diff
@@ -444,7 +444,11 @@ public void serverTag(ReplaceableTagEvent event) {
         // Returns a list of all files in the specified directory. The starting path is /plugins/Denizen.
         // -->
         if (attribute.startsWith("list_files") && attribute.hasContext(1)) {
-            File[] files = DenizenAPI.getCurrentInstance().getDataFolder().listFiles();
+            File folder = new File(DenizenAPI.getCurrentInstance().getDataFolder(), attribute.getContext(1));
+            if (!folder.exists()) {
```
Comment (Morphan1) [inline src/main/java/net/aufdemrand/denizen/tags/core/ServerTags.java]: || !folder.isDirectory()

File: src/main/java/net/aufdemrand/denizen/scripts/commands/item/NBTCommand.java [hunk 2]
```diff
@@ -56,13 +56,13 @@ public void execute(ScriptEntry scriptEntry) throws CommandExecutionException {
...
-        dB.log("testing");
+//        dB.log("testing");
         if (value.asString().equals("")) {
-            dB.log("removing");
+//            dB.log("removing");
             itemStack = CustomNBT.removeCustomNBT(itemStack, key.asString());
         }
         else {
-            dB.log("adding");
+//            dB.log("adding");
```
Comment (Morphan1) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/item/NBTCommand.java]: This was already removed, most likely what's causing merge conflicts

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 3]
```diff
@@ -2139,6 +2139,10 @@ public void registerCoreMembers() {
         // Use to edit the NBT of a player's item in hand.
         // - nbt <player.item_in_hand> "MyCustomNBT.Owner:<player>" "save:edited"
         // - inventory set "slot:<player.item_in_hand.slot>" "o:<entry[edited].new_item>"
+        // @Usage
+        // Use to remove an NBT tag from a player's item in hand.
+        // - nbt <player.item_in_hand> "MyCustomNBT.Owner:" "save:item"
+        // - inventory set "slot:<player.item_in_hand.slot>" "o:<entry[item].new_item>"
```
Comment (Morphan1) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: I actually... don't think I meant to make it that way, I think I meant to go back after I decided whether it should be "!" or something else. @mcmonkey4eva what do you think about this functionality?

File: src/main/java/net/aufdemrand/denizen/events/world/LingeringPotionSplashScriptEvent.java [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+import net.aufdemrand.denizencore.utilities.CoreUtilities;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.AreaEffectCloud;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.LingeringPotionSplashEvent;
+
+public class LingeringPotionSplashScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // TODO: Fix Event (Maybe spigot)
```
Comment (Morphan1) [inline src/main/java/net/aufdemrand/denizen/events/world/LingeringPotionSplashScriptEvent.java]: What does this mean?

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java (hunk 3 above)
Comment (BlackCoyote) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: it should definitely be ! just for consistency sake

File: src/main/java/net/aufdemrand/denizen/events/world/LingeringPotionSplashScriptEvent.java (hunk 4 above)
Comment (Fortifier42) [inline src/main/java/net/aufdemrand/denizen/events/world/LingeringPotionSplashScriptEvent.java]: I was completely unable to get the event to fire, i.e. the bukkit event wasn't firing. I added debug to test and that's what I came up with

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java (hunk 3 above)
Comment (Fortifier42) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: I've done this now :)

File: src/main/java/net/aufdemrand/denizen/tags/core/ServerTags.java (hunk 1 above)
Comment (Fortifier42) [inline src/main/java/net/aufdemrand/denizen/tags/core/ServerTags.java]: Fixed this too :heart:

**Pull request merged**